src/main/scala/scala/tools/partest/sbt/SBTRunner.scala

```diff
@@ -6,12 +6,15 @@
 **                          |/                                          **
 \*                                                                      */
 
-package scala.tools.partest
-package nest
+package scala.tools.partest.sbt
```

Comment:
who is using this class? can we easily just move it and update its users simultaneously?